src/test/java/co/rsk/federate/signing/hsm/message/AdvanceBlockchainMessageTest.java

```diff
@@ -27,36 +28,44 @@ public class AdvanceBlockchainMessageTest {
     public void setUp() {
         blockHeaderBuilder = new BlockHeaderBuilder(mock(ActivationConfig.class));
 
-        BlockHeader block1Header = blockHeaderBuilder.setNumber(1).build();
-        BlockHeader block2Header = blockHeaderBuilder.setNumber(2).build();
-        BlockHeader block3Header = blockHeaderBuilder.setNumber(3).build();
-
-        List<BlockHeader> block1Brothers = Arrays.asList(
-            blockHeaderBuilder.setNumber(101).build(),
-            blockHeaderBuilder.setNumber(102).build()
+        List<BlockHeader> block1Uncles = Arrays.asList(
```

Comment:
Could even copy the buildBlocks method from HsmBookkeepingClientImplTest, it will make it clearer and easier to follow